brittany/Dockerfile

```diff
@@ -1,3 +1,3 @@
-FROM debian:10.6-slim
-COPY ghc-lib/ /opt/ghc/8.10.2/lib/ghc-8.10.2/
+FROM amazonlinux:2.0.20200722.0
+COPY ghc-lib/ /opt/ghc/8.10.3/lib/ghc-8.10.3/
```

Comment:
We may need to upgrade these directories (away from `/opt/ghc/...`) in order to get things actually working.